src/ensembl/src/shared/components/in-app-search/InAppSearch.tsx

```diff
@@ -109,6 +109,9 @@ const InAppSearch = (props: Props) => {
             onSubmit={onSearchSubmit}
             className={styles.searchField}
             rightCorner={rightCorner}
+            onKeyUp={(e) => e.stopPropagation()}
+            onKeyDown={(e) => e.stopPropagation()}
+            onKeyPress={(e) => e.stopPropagation()}
```

Comment:
What's the purpose of these event listeners and of stopping the event propagation?

Reply:
Dan says version 0.0.11 of the genome browser should only respond to rare combinations of keypresses. Let's update to 0.0.11 and remove these listeners.

Reply:
That's done